source/_docs/regions.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Create a New Site in Pantheon's EU Region
-description: Learn how to get Limited Availability access to and launch sites in Pantheon's European Union Region.
+title: Create a New Site in your Choice of Pantheon Region
+description: Learn how to get Limited Availability access to and launch sites in a specific Pantheon Region.
```

Comment:
there are only 4 regions planned, so I think it makes sense to spell them all out.... "launch sites in the European Union or Australia."